server/verify_test.go

```diff
@@ -358,7 +370,7 @@ func TestVerifyBasicAttestationWithTdx(t *testing.T) {
 	}
 	defer ak.Close()
 
-	nonce := []byte("super secret nonce")
+	nonce := tdxReportData
```

Comment:
I don't know the TDX specifics very well. Does "reportData" usually refer to a nonce?

Reply:
Yes!